src/ansible_compat/config.py

```diff
@@ -78,18 +78,12 @@ def ansible_version(version: str = "") -> Version:
         stdout=subprocess.PIPE,
         stderr=subprocess.PIPE,
     )
-    if proc.returncode == 0:
-        version, error = parse_ansible_version(proc.stdout)
-        if error is not None:
-            print(error)
-            raise MissingAnsibleError()
-    else:
-        print(
-            "Unable to find a working copy of ansible executable.",
-            proc,
+    if proc.returncode != 0:
+        raise MissingAnsibleError(
+            "Unable to find a working copy of ansible executable.", proc=proc
```

Comment:
Why did you drop a trailing comma here?

Reply:
I was not me, it was black. Commas are useful only when you have multiple lines so you can easily add new entry or reorder them.